### PR TITLE
feat: Check Verifiable Credential implementation against Verifiable Claims WG Test Suite

### LINKED ISSUE #326: Verifiable Credential compatibility with Verifiable Claims WG Test Suite
We need to make sure that the Verifiable Credential implementation conforms to Verifiable Claims Working Group test suite (https://github.com/w3c/vc-test-suite).

### PR DESCRIPTION
**Pull request by kdimak**
Add makefile target and a script which checkouts and setup Verifiable Claims WG Test Suite. Implement Test Suite generator which uses aries-framework-go Verifiable Credential API. The suite tests the Verifiable Credential API compatibility with the specification via the generator.

closes #326

Signed-off-by: Dima <dkinoshenko@gmail.com>

### CODE REVIEW

File: pkg/doc/verifiable/test_suite/verifiable_suite_test.go [hunk 1]
```diff
@@ -0,0 +1,162 @@
+/*
```
Comment (troyronda) [inline pkg/doc/verifiable/test_suite/verifiable_suite_test.go]: Underscore in package name is a bit unusual.

File: scripts/run_vc_test_suite.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+cd $VC_TEST_SUITE
+git clone --depth=1 https://github.com/w3c/vc-test-suite
+
+# build the suite
+cd vc-test-suite
+npm install
+cp "${DIR}/pkg/doc/verifiable/test_suite/config.json" .
+
+# run the suite
+#npm test -- --timeout 10000
```
[the remark this answers is not in the document]
Comment (kdimak) [reply scripts/run_vc_test_suite.sh]: Yeah, I will drop it and also add a paragraph to the README where I can explain how to run and report results to the community.

File: pkg/doc/verifiable/test_suite/verifiable_suite_test.go (hunk 1 above)
Comment (kdimak) [reply pkg/doc/verifiable/test_suite/verifiable_suite_test.go]: Will be fixed!

Comment (kdimak) [reply pkg/doc/verifiable/test_suite/verifiable_suite_test.go]: Fixed.

File: scripts/run_vc_test_suite.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (kdimak) [reply scripts/run_vc_test_suite.sh]: Cleaned up, also added README section.

File: README.md [hunk 3]
```diff
@@ -54,6 +54,18 @@ It can be generated by running command `make generate-openapi-test`
 Generated spec can be found under 
 `build/rest/openapi/spec`
 
+##### Verifiable Credential Test Suite
+In order to check compatibility of Verifiable Credential implementation in aries-framework-go with 
+[W3C Verifiable Claims Working Group Test Suite](https://github.com/w3c/vc-test-suite), run `make vc-test-suite`.
+The result of Test Suite run is placed under `vc-test-suite/suite/implementations/aries-framework-go-report.json`.
+In order to public the results to the community, run:
```
Comment (troyronda) [inline README.md]: I don't think we need lines 61-67 in the README

File: README.md [hunk 4]
```diff
@@ -54,6 +54,18 @@ It can be generated by running command `make generate-openapi-test`
 Generated spec can be found under 
 `build/rest/openapi/spec`
 
+##### Verifiable Credential Test Suite
+In order to check compatibility of Verifiable Credential implementation in aries-framework-go with
```
Comment (troyronda) [inline README.md]: To test compatibility of the verifiable credential packages with ...

File: README.md [hunk 5]
```diff
@@ -54,6 +54,18 @@ It can be generated by running command `make generate-openapi-test`
 Generated spec can be found under 
 `build/rest/openapi/spec`
 
+##### Verifiable Credential Test Suite
+In order to check compatibility of Verifiable Credential implementation in aries-framework-go with 
+[W3C Verifiable Claims Working Group Test Suite](https://github.com/w3c/vc-test-suite), run `make vc-test-suite`.
+The result of Test Suite run is placed under `vc-test-suite/suite/implementations/aries-framework-go-report.json`.
```
Comment (troyronda) [inline README.md]: The result of the test suite is generated as ...

File: pkg/doc/verifiable/test-suite/verifiable_suite_test.go [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+func main() {
+	inputFile := os.Args[len(os.Args)-1]
+	vcBytes, readErr := ioutil.ReadFile(inputFile) // nolint:gosec
+	if readErr != nil {
+		log.Println(fmt.Errorf("cannot open input file %s: %w", inputFile, readErr))
+		abort()
+	}
+
+	jwt := flag.String("jwt", "", "base64encoded JSON object containing es256kPrivateKeyJwk and rs256PrivateKeyJwk.")
+	// todo use jwtAud
```
Comment (troyronda) [inline pkg/doc/verifiable/test-suite/verifiable_suite_test.go]: issue number?

File: README.md (hunk 4 above)
Comment (kdimak) [reply README.md]: Thanks! I will fix that.

File: README.md (hunk 3 above)
Comment (kdimak) [reply README.md]: OK

File: README.md (hunk 5 above)
Comment (kdimak) [reply README.md]: Thanks!

File: pkg/doc/verifiable/test-suite/verifiable_suite_test.go (hunk 6 above)
Comment (kdimak) [reply pkg/doc/verifiable/test-suite/verifiable_suite_test.go]: #371

File: README.md (hunk 3 above)
Comment (kdimak) [reply README.md]: OK

File: README.md (hunk 4 above)
Comment (kdimak) [reply README.md]: Fixed

File: README.md (hunk 5 above)
Comment (kdimak) [reply README.md]: Corrected.

**Pull request merged**